app/src/main/java/ch/epfl/sdp/drone3d/storage/data/MappingMission.kt

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.sdp.drone3d.storage.data
+
+enum class State {
+    NOT_STORED, PRIVATE, SHARED, PRIVATE_AND_SHARED
+}
+
+data class MappingMission(
```

Comment:
This lacks a little bit of documentation.

What is the state for, what are the id mutable ?

Reply:
Better ?